src/test/java/io/cdap/plugin/http/source/common/HttpBatchSourceConfigTest.java

```diff
@@ -0,0 +1,21 @@
+package io.cdap.plugin.http.source.common;
+
+import io.cdap.plugin.http.source.batch.HttpBatchSourceConfig;
+import org.junit.Test;
+
+/**
+ * Unit tests for HttpBatchSourceConfig
+ */
+public class HttpBatchSourceConfigTest {
+    private static final String MOCK_STAGE = "mockStage";
+
+    private static final HttpBatchSourceConfig CONFIG = new HttpBatchSourceConfig("test",
+            "http://localhost", "GET", "Auth:", "JSON", "false",
+            "", "", 600L, 120, 120,
+            "NONE", "true");
+
+    @Test (expected = IllegalArgumentException.class)
+    public void testValidConfig() {
```

Comment:
Please rename to something like testMissingKeyValue

Reply:
done